sites/svelte-5-preview/src/routes/docs/content/03-appendix/02-breaking-changes.md

```diff
@@ -9,17 +9,21 @@ While Svelte 5 is a complete rewrite, we have done our best to ensure that most
 In Svelte 3 and 4, components are classes. In Svelte 5 they are functions and should be instantiated differently. If you need to manually instantiate components, you should use `mount` or `createRoot` (imported from `svelte`) instead. If you see this error using SvelteKit, try updating to the latest version of SvelteKit first, which adds support for Svelte 5. If you're using Svelte without SvelteKit, you'll likely have a `main.js` file (or similar) which you need to adjust:
 
 ```diff
-+ import { mount } from 'svelte';
++ import { createRoot } from 'svelte';
 import App from './App.svelte'
 
 - const app = new App({ target: document.getElementById("app") });
-+ const app = mount(App, { target: document.getElementById("app") });
++ const app = createRoot(App, { target: document.getElementById("app") });
 
 export default app;
 ```
 
+`createRoot` returns an object with a `$set` and `$destroy` method on it. It does not come with an `$on` method you may know from the class component API. Instead, pass them via the `events` property on the options argument. If you don't need to interact with the component instance after creating it, you can use `mount` instead, which saves some bytes.
```

Comment:
Is this right? `createRoot` doesn't take an `events` option, it's all just props

Reply:
It does take one, and for backwards compatibility it _needs_ to take one. If you want to instantiate a component using the old `createEventDispatcher` and pass in listeners for those events, you need to pass it via the `events` property.

We could add something like `(note that passing events via the events property is deprecated, you should use [event attributes instead](link))` to make sure people understand the nuances.